README.md

```diff
@@ -1 +1,118 @@
 # fabric8-analytics-notification-scheduler
+
+# Developing fabric8-analytics-notification-scheduler
+
+This section is for those interested in contributing to the development of
+fabric8-analytics-notification-scheduler.
+
+## Running a Local Instance
+
+### Getting fabric8-analytics-notification-scheduler repository
+
+### Requirements
+
+Git, and possibly other packages, depending on how you want to run the system
+(see below).
+
+### Getting the Code
+
+First of all, clone the `fabric8-analytics-notification-scheduler` repo (this one).
+This includes all the configuration for running the whole system as well as some helper
+scripts and docs.
+
+### Running via docker
+
+Requirements:
+
+* docker >= 1.10.0
```

Comment:
Why do we need at least this version (?)

Reply:
@msrb, I think most new features are available after this version. I can remove this line if you say. :)

Reply:
Please, remove it :) I think we have something similar in worker repo as well, but the docs are quite outdated.

Reply:
@msrb, sure :)